instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Comment from './Comment';
+import './CommentSection.css';
+
+CommentSection.propTypes = {
+    postData: PropTypes.shape({
+        comments: PropTypes.array,
+        timestamp: PropTypes.string
+    })
+};
+
+class CommentSection extends React.Component {
+
+    state = {
+        comments: this.props.post.comments,
```

Comment:
In PostContainer, you named your prop containing the comments array 'comments', but you're trying to use `this.props.post` in this file where you didn't pass down a `post` prop. It should be `this.props.comments`.